### DISCUSSION TITLE
Add Symbol Lookup/Panel

### DISCUSSION THREAD

**Discussion by BrandowLucas** [community/community#136670] [Mobile]
When tapping on a function or variable in the code, the app should locate its declaration or definition by showing the Symbol Panel pop-up. This feature is available in the browser version but is not yet supported in the mobile app.

**Comment by eliperkins** [comment]
Hey @BrandowLucas, thanks for this feedback! I definitely agree that navigating code by tapping on symbols, along with a symbol panel, would be quite helpful in getting around a codebase. I've created an internal tracking issue for this feature request. I'll work with the team to figure out what's needed here to get symbol navigation implemented.

Thanks for helping improve GitHub Mobile!
